correlation.go

```diff
@@ -0,0 +1,46 @@
+// Licensed to SkyAPM org under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. SkyAPM org licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package go2sky
+
+import "context"
+
+func PutCorrelation(ctx context.Context, key, value string) bool {
+	activeSpan := ctx.Value(ctxKeyInstance)
+	if activeSpan == nil {
+		return false
+	}
+
+	span, ok := activeSpan.(segmentSpan)
+	if ok {
+		span.context().CorrelationContext[key] = value
```

Comment:
Take a look at Java's `Config.Correlation.*`. This propagation is not unlimited for keeping the whole system safe.

Reply:
Should I add a `opt` to build the tracer? such as `WithCorrelation(keyCount, valueSize int)` ?

Reply:
Yes, for go sdk, configs are all from there, until we support CDS later.

Reply:
Added. please take a look.